### PR TITLE
MinGW: Change some of the _MSC_VER to _WIN32

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

File: Source/App/EbAppConfig.h [hunk 1]
```diff
@@ -10,309 +10,285 @@
...
-
-    uint64_t        processed_frame_count;
-    uint64_t        processed_byte_count;
-
+typedef struct EbPerformanceContext {
+
+  /****************************************
+   * Computational Performance Data
+   ****************************************/
+  uint64_t lib_start_time[2];    // [sec, micro_sec] including init time
```
Comment (tianjunwork) [inline Source/App/EbAppConfig.h]: Majority code use 4 space indent. Could you also use 4 space? Thx.

File: Source/Lib/Codec/EbEncHandle.c [hunk 2]
```diff
@@ -2983,7 +2983,7 @@ static EbErrorType init_svt_vp9_encoder_handle(
 #endif
 
 #ifndef _RSIZE_T_DEFINED
-typedef size_t Rsize;
+typedef size_t rsize_t;
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbEncHandle.c]: from AV1?

Comment (tianjunwork) [reply Source/Lib/Codec/EbEncHandle.c]: already defined in Ebdefinition.h?

File: Source/App/EbAppConfig.h (hunk 1 above)
Comment (1480c1) [reply Source/App/EbAppConfig.h]: oops, clang format used default, going to revert

File: Source/Lib/Codec/EbEncHandle.c (hunk 2 above)
Comment (1480c1) [reply Source/Lib/Codec/EbEncHandle.c]: Rsize was seeming to give error when compiling with mingw, using rsize_t and replacing Rsize worked fine

error:
``` none
In file included from ../Source/Lib/VPX/vpx_dsp_rtcd.h:33,
                 from ../Source/Lib/VPX/loopfilter.c:13:
../Source/Lib/Codec/EbDefinitions.h:423:8: error: unknown type name 'Rsize'
  423 | extern Rsize
      |        ^~~~~
../Source/Lib/Codec/EbDefinitions.h:424:27: error: unknown type name 'Rsize'; did you mean 'rsize_t'?
  424 | strnlen_ss(const char *s, Rsize smax);
      |                           ^~~~~
      |                           rsize_t
```

**Pull request merged**